static/app/views/nav/secondary/sections/issues/issueViews/issueViewItem.tsx

```diff
@@ -226,6 +226,7 @@ const TrailingItemsWrapper = styled('div')`
 const StyledSecondaryNavItem = styled(SecondaryNav.Item)`
   position: relative;
   padding-right: ${space(0.5)};
+  gap: ${space(0.5)};
```

Comment:
Can you do this in the SecondaryNav.Item so that it looks this way across all surfaces?